### PR TITLE
REPLAY-584: Add a guide on allowing cookie blocking exception

### PR DESCRIPTION
**Pull request by acorretti**
### What does this PR do?

Adds a RUM Session Replay FAQ page describing how to make an exception to 3rd-party cookie blocking, in order to allow Session Replay's upcoming Service Worker functionality.

### Motivation

Service Worker is an upcoming feature that will provide a more performant replaying experience. If the user's browser is blocking 3rd-party cookies, this will also block our service worker. Session Replay's player page will detect this situation and prompt the user to make an exception, showing a link to this page to read more.

Jira card: https://datadoghq.atlassian.net/browse/REPLAY-584

### Preview

https://docs-staging.datadoghq.com/alfonso.corretti/REPLAY-584-allow-third-party-service-worker/real_user_monitoring/faq/session_replay_service_worker/

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/real_user_monitoring/faq/session_replay_service_worker.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+kind: faq
+further_reading:
+    - link: '/real_user_monitoring/session_replay/'
+      tag: 'Get Started'
+      text: 'Session Replay'
+---
+
+## Third-Party Cookie Blocking And Session Replay
+
+Session Replay uses a *Service Worker* in a different domain (*session-replay-datadoghq.com*) to provide the best experience possible whilst protecting your privacy and guaranteeing the safety of your data.
```
Comment (alai97) [inline content/en/real_user_monitoring/faq/session_replay_service_worker.md:12]:
```suggestion
## Overview

Session Replay uses a Service Worker in another domain `session-replay-datadoghq.com` to provide the best experience possible while protecting your privacy and guaranteeing the safety of your data.

{{< img src="real_user_monitoring/session_replay/allow-3p-serviceworker-1.png" alt="Allow Session Replay Third-Party Service Worker" >}}
```

File: content/en/real_user_monitoring/faq/session_replay_service_worker.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+---
+title: Allow Third-Party Service Worker for Session Replay
+kind: faq
+further_reading:
+    - link: '/real_user_monitoring/session_replay/'
+      tag: 'Get Started'
+      text: 'Session Replay'
+---
+
+## Third-Party Cookie Blocking And Session Replay
```
Comment (alai97) [inline content/en/real_user_monitoring/faq/session_replay_service_worker.md:10]:
```suggestion
```

File: content/en/real_user_monitoring/faq/session_replay_service_worker.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+Session Replay uses a *Service Worker* in a different domain (*session-replay-datadoghq.com*) to provide the best experience possible whilst protecting your privacy and guaranteeing the safety of your data.
+
+If you have blocked third-party cookies in your browser settings, or if your browser blocks them by default, this can prevent the service worker from registering correctly.
```
Comment (alai97) [inline content/en/real_user_monitoring/faq/session_replay_service_worker.md:14]:
```suggestion
If you have blocked third-party cookies in your browser settings, or if your browser blocks them by default, this prevents the service worker from registering your replays correctly.
```

File: content/en/real_user_monitoring/faq/session_replay_service_worker.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+### Allow an exception
+
+Datadog's recommended solution is to make an exception to your third-party cookie blocking, to allow Session Replay's service worker to work and enjoy the most performant replaying experience:
```
Comment (alai97) [inline content/en/real_user_monitoring/faq/session_replay_service_worker.md:18]:
```suggestion
Datadog recommends that you make an exception to your third-party cookie blocking to allow Session Replay's service worker to function properly.
```

File: content/en/real_user_monitoring/faq/session_replay_service_worker.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+{{< img src="real_user_monitoring/session_replay/allow-3p-serviceworker-1.png" alt="Allow Session Replay Third-Party Service Worker" >}}
```
Comment (alai97) [inline content/en/real_user_monitoring/faq/session_replay_service_worker.md:20]: Moving this image to the section above. 

```suggestion
```

**Review by alai97 — CHANGES_REQUESTED** [review]
Documentation review!

File: content/en/real_user_monitoring/faq/session_replay_service_worker.md (hunk 3 above)
Comment (acorretti) [reply content/en/real_user_monitoring/faq/session_replay_service_worker.md:14]: ... from registering **your replays** correctly.

It isn't registering _replays_, but it's the registration of the Service Worker itself. Just this change is not technically accurate.

File: content/en/real_user_monitoring/faq/session_replay_service_worker.md (hunk 5 above)
Comment (acorretti) [reply content/en/real_user_monitoring/faq/session_replay_service_worker.md:20]: The two screenshots are steps to make the cookie exception, I'm not sure it makes sense to place them separately. Should I run this through Design to see if there's a chance to improve the artwork?

Comment (julie2711) [reply content/en/real_user_monitoring/faq/session_replay_service_worker.md:20]: Yes I agree with Alfonso, the screenshots are closely related steps to reactivate 3rd party cookies for DD and should not be separated; to make it clearer than they are in fact steps the user can/should follow, i would suggest to add a small line / paragraph indicating that these screenshots are part of the process to follow.

File: content/en/real_user_monitoring/faq/session_replay_service_worker.md (hunk 3 above)
Comment (alai97) [reply content/en/real_user_monitoring/faq/session_replay_service_worker.md]: Thanks for the additional feedback!

File: content/en/real_user_monitoring/faq/session_replay_service_worker.md (hunk 5 above)
Comment (alai97) [reply content/en/real_user_monitoring/faq/session_replay_service_worker.md:20]: Ahh, that sounds good! I didn't know they were in order. Let's add some instructions in then 😛

**Review by alai97 — APPROVED** [review]
Added instructions to complement the screenshots!

**Comment by acorretti** [comment]
All changes ready, @alai97 you can do a final review & then merge

**Pull request merged**